bundles/org.openhab.binding.tplinkrouter/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="tplinkrouter" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>TpLinkRouter Binding</name>
```

Comment:
Entry to be removed (it is no more present in any bindings since a long time).

Reply:
Name is still used. Aren't you mixing up with author tag here, which is not used anymore?

Reply:
Oh yes, you are right, sorry for this stupid comment.